tensorflow/python/ops/init_ops_test.py

```diff
@@ -58,114 +59,127 @@ def _runner(self,
       self.assertGreater(lim, abs(output.min() - target_min))
 
   def test_uniform(self):
-    tensor_shape = (9, 6, 7)
+    shape = [9, 6, 7]
     with self.cached_session():
-      self._runner(
-          init_ops.RandomUniform(minval=-1, maxval=1, seed=124),
-          tensor_shape,
-          target_mean=0.,
-          target_max=1,
-          target_min=-1)
+      for tensor_shape in [shape, tensor_shape_lib.TensorShape(shape)]:
```

Comment:
@fchollet Thanks for your feedback. I'm just a little confused, because I think new test cases of TensorShape for init_ops have been added. I have no idea of what new test case is needed, would you mind giving me more clues?

Reply:
By the way, do we need to delete the unit test abut keras layers ?

Reply:
I have deleted the unit test in keras, and only preserved those unit tests for init_ops.